mobile/android/scripts/download-libnode.sh

```diff
@@ -0,0 +1,18 @@
+# Run from mobile/android
+
+# Check if libnode already exists
+if [ -f "./app/src/main/cpp/libnode/bin/arm64-v8a/libnode.so" ]; then
+    echo "libnode already exists, skipping download."
+    exit 0
+fi
+
+DOWNLOAD_URL="https://github.com/mustang-im/nodejs-mobile/releases/download/v22.9.0/nodejs-mobile-v22.9.0-android.zip"
```

Comment:
In general, downloading binary blobs is uncool and dangerous. It will get is thrown out of F-Droid (or rejected from), for instance.
It's also against reproducible builds.